de.syping.gta5view.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This belongs upstream, so you can also updated `<releases>` section as part of your release procedure. The file is not specific to Flathub and other "stores" (Linux distros or snap) can also use it.